specification/columns/chargeclassification.md

```diff
@@ -0,0 +1,42 @@
+# Charge Classification
+
+A Charge Classification indicates whether the row represents a regular charge or a correction to one or more previous charges.
+
+
+The ChargeClassification column MUST be present and MUST NOT be null. This column is of type String and MUST be one of the allowed values.
+
+## Column ID
+
+ChargeClassification
+
+## Display Name
+
+Charge Classification
+
+## Description
+
+Indicates whether the row represents a regular charge or a correction to one or more previous charges.
+
+
+## Content Constraints
+
+| Constraint      | Value          |
+| :-------------- | :------------- |
+| Column type     | Dimension      |
+| Column required | True           |
+| Allows nulls    | False          |
+| Data type       | String         |
+| Value format    | Allowed values |
+
+Allowed values:
+
+| Value      | Description                          |
+| :--------- | :------------------------------------|           |
+| Regular     | Normal charge  |

```

Comment:
As I insinuated above, just having "Regular" and "Correction" seems too abstract

Reply:
I mean a good alternative we had is outlined in the supporting content option 3 i don't know if you feel this is any better?
https://github.com/FinOps-Open-Cost-and-Usage-Spec/FOCUS_Spec/blob/330-Charge-Category-Credit-Refund/supporting_content/columns/chargecategory.md#option-3

Reply:
Hey @cnharris10 following the discussion on 23.3 are you ok with the amendments in the PR

Reply:
Some potential alternatives:
- Standard (may be confused with PricingCategory)
- Baseline
- Base
- Original
- Conventional

Reply:
Hey this adjustment was made and approved for merge on the previous joint WG call, these suggestions could be opened as an issue